students/21th/jiminkim/js/main.js

```diff
@@ -0,0 +1,158 @@
+const loginId = `jemizem`;
+
+
+const txtFindPoeple = document.getElementById('txtFindPoeple');
+
+var searchBox = document.getElementById('head-search-wrap');
+const ulElement = document.getElementById('ulFindedPeopleList');
+
+const appendComment = (event) => {
+    
+    let commentId = event.target.id.substr(-2);
+    const commentTextarea = document.getElementById('textComment'+commentId);
+    const commentAddButton = document.getElementById('commentButton'+commentId);
+    const commentList = document.getElementById('commentList'+commentId);
+
+    let commentStr = commentTextarea.value;
+    commentStr = commentStr.replace(/(?:\r\n|\r|\n)/g,'');
+
+    if (commentStr == '' ) {
+        commentAddButton.classList.add("inactive");
+        return;
+    } else {
+        commentAddButton.classList.remove("inactive");
+    }
+
+    if (event.target.nodeName === 'TEXTAREA') {
+        if (event.key != 'Enter') return;
+    }
+
+    let commentLi = document.createElement('li')
+    commentList.appendChild(commentLi);
+
+    // 댓글 element 만들기 프로필사진 / 아이디 / 댓글내용 / 삭제 / 하트
+    let commenter = document.createElement('span');
+    commenter.innerHTML = `${loginId}`;
+    commenter.className = "nickname bold";
+    commentLi.appendChild(commenter);
+
+    let commentContent = document.createElement('span');
+    commentContent.innerHTML = commentTextarea.value;
+    commentContent.className = "comment";
+    commentLi.appendChild(commentContent);
+    commentTextarea.value = "";
+    
+    let commentButtonDiv = document.createElement('div');
+    commentButtonDiv.className = "button-list";
+    commentLi.appendChild(commentButtonDiv);
+
+    let commentDeleteButton = document.createElement('button');
+    commentDeleteButton.innerHTML = "✖️";
+    commentDeleteButton.className = "comment-delete-button";
+    commentDeleteButton.addEventListener('click',(event)=>{ 
+        const parentUl = event.target.parentElement.parentElement.parentElement;
+        const deleteLi = event.target.parentElement.parentElement;
+        parentUl.removeChild(deleteLi);
+    });
```

Comment:
```suggestion
    commentDeleteButton.addEventListener('click',()=>{ 
        commentLi.remove();
    });
```
와 같이 도 가능합니다! 참고해주세요~
- `commentLi` 는 `appendChild` 함수가 호출될 때마다 새로 생성되는 요소이기 때문에, `commentLi` 라고만 해도 다른 `commentLi` 과 구분이 됩니다.